src/pipedream/errors/not_found_error.py

```
@@ -0,0 +1,10 @@
+# This file was auto-generated by Fern from our API Definition.
+
+import typing
+
+from ..core.api_error import ApiError
+
+
+class NotFoundError(ApiError):
+    def __init__(self, body: typing.Any, headers: typing.Optional[typing.Dict[str, str]] = None):
```

Ruff (0.14.8)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)